packages/dropdowns/src/elements/Dropdown/Dropdown.spec.tsx

```diff
@@ -12,7 +12,7 @@ import { Dropdown, Trigger, Menu, Item, NextItem, PreviousItem } from '../..';
 import { IDropdownProps } from './Dropdown';
 
 const ExampleDropdown = (props: IDropdownProps) => (
-  <Dropdown {...props}>
+  <Dropdown data-test-id="dropdown" {...props}>
```

Comment:
As a rule, we do not provide external test IDs – they are stripped by the build config. This is left as an exercise for the consumer in their own component test setup.